native-rating/implementation.md

```diff
@@ -0,0 +1,212 @@
+---
+title: Building an Implementation
+---
+
+## Building a Native Rating implementation
+
+### Install the CLI
+
+First, ensure that you have [Node](https://nodejs.org/en/) v12+ and [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) installed. Mac users will also need the development tools installed, but you will be prompted to install them
+when you run the CLI.
+
+To install [ShipEngine Connect CLI](https://connect.shipengine.com/), run the following command from your terminal:
+
+```bash
+npm install --global @shipengine/connect
+```
+
+Verify that the CLI was installed correctly:
+
+```bash
+connect --version
+```
+
+### Create the Native Rating app
+
+The Connect CLI can create a Native Rating app for you by running the following from your command line,
+replacing `my-native-rating-app` with the path you want to use for the app:
+
+```bash
+mkdir my-native-rating-app
+cd my-native-rating-app
+connect init
+```
+
+When this is finished, you will have a "fully working" Native Rating connect app, ready to be modified however you see fit.
+
+## Configuration
+
+Before building your implementation, it is recommended to get the basic configuration out of the way.
+
+### AppID
+
+The appId is a unique identifier required by Connect and cannot change after the app is first published. The CLI will
+creates a random GUID and it is located in the `appId` property in `package.json` at the root of the app package.
```

Comment:
Either use `The CLI will create...` or `The CLI creates...`

Reply:
updated